### DISCUSSION TITLE
make an educational tool

### DISCUSSION THREAD

**Discussion by sebschopf** [phcode-dev/phoenix#1012] [Ideas]
Hello,

My English is very poor in writing, so I'm using DeepL to write this message (please don't hold it against me).

I just discovered phoenix through Brackets, which I use to experiment things.

I'm not very good at using HTML and CSS. I have to say I'm learning from popular online sites like Flexbox Froggy, CSS grid garden and w3schools.com and lately I've been asking questions at GPT (developer.mozilla.org also helps but it's not very educational, I see it as an encyclopedia, gpt help).

It seems to me that phoenix is amazing. I don't know the structure behind the project, so I'm wondering if you could create exercises from the very beginning (how to structure a project, tree structure, etc., technology) to then create project scenarios that will go from a presentation page to "anything more complex" by integrating concepts each time?
I'm thinking that with phoenix, it would be cool to follow a lot of stuff and learn step by step through the creation of a site (even the introduction of framwork).
At the moment I can't even learn javascript, because I'm not even sure of its possibilities.

Anyway, I think it's an off-topic idea regarding this project, but here it is, I'm just throwing it out there, hoping that maybe it will stick in someone's head.

Anyway, congratulations on the final result. I will certainly talk about it around me ;)

Seb.

Translated with www.DeepL.com/Translator (free version)

**Comment by abose** [comment]
Thanks for the feature request @sebschopf . We also like the idea of having better learning tools integrated into phcode. We have toyed with the idea, and it is on our list of things of interest. Automated BootCamp for coding for beginners fits right into our student users needs.

Created feature request https://github.com/phcode-dev/phoenix/issues/1013 . This may not happen in the next 6 months as our hands are full with native windows/mac/Linux builds and general code intelligence improvements.
